Fix build & install on osx (#31)
libmpack/libmpack-lua: README.md

[![Travis Build Status](https://travis-ci.org/libmpack/libmpack-lua.svg?branch=master)](https://travis-ci.org/libmpack/libmpack-lua)

## Building

```bash
LUA_TARGET=$(PLATFORM) make
```

> Where `PLATFORM` is a supported Lua 5.1 platform: linux (default), freebsd, macosx, ...
> For a complete list of targets run:

Reviewer: ```suggestion
Where `PLATFORM` is a supported Lua 5.1 platform: linux (default), freebsd, macosx, ...

For the complete list of targets run:
```

[![Travis Build Status](https://travis-ci.org/libmpack/libmpack-lua.svg?branch=master)](https://travis-ci.org/libmpack/libmpack-lua)

## Building

```bash
LUA_TARGET=$(PLATFORM) make
```

> Where `PLATFORM` is a supported platform for lua 5.1 (freebsd, linux, macosx etc.)
> 
> ```bash
> # e.g. for osx
> LUA_TARGET=macosx make
> ```
> 

Reviewer: ```suggestion
Where `PLATFORM` is a supported Lua 5.1 platform: linux (default), freebsd, macosx, ...
```

[![Travis Build Status](https://travis-ci.org/libmpack/libmpack-lua.svg?branch=master)](https://travis-ci.org/libmpack/libmpack-lua)

## Building

```bash
LUA_TARGET=$(PLATFORM) make
```

Where `PLATFORM` is a supported platform for lua 5.1 (freebsd, linux, macosx etc.)

```bash
# e.g. for osx
LUA_TARGET=macosx make
```

For a complete list of targets run:
```bash
make; cd .deps/5.1.5/src/lua && make
```

> 
> `LUA_TARGET` will default to `linux` if not specified.

Reviewer: ```suggestion
```

[![Travis Build Status](https://travis-ci.org/libmpack/libmpack-lua.svg?branch=master)](https://travis-ci.org/libmpack/libmpack-lua)

## Building

```bash
LUA_TARGET=$(PLATFORM) make
```

Where `PLATFORM` is a supported platform for lua 5.1, i.e. one of:
  * aix
  * ansi
  * bsd
  * freebsd
  * generic
  * linux
  * macosx
  * mingw
  * posix

>   * solaris

Reviewer: This is a lot of space dedicated to a pretty niche use-case. Is there maybe a Lua source you can just link to instead of listing everything here? Or perhaps just list this in parens instead of a bullet list.

Author: I'll update it to put them in parenthesis (likely tomorrow morning).

I tried finding a source to link to but I can't find one for lua 5.1 and the relevant page for [lua 5.4](https://www.lua.org/manual/5.4/readme.html) lists targets that aren't available in 5.1 (e.g. `guess`)

Author: I could also drop the full list, just give one example, and tell users to `cd .deps/src/5.1/lua && make` for a full list of platforms

Author: @justinmk updated the instructions to not devote so much space to niche platforms.

LMK if you'd like the same treatment for https://github.com/neovim/lua-client and I'll happily put together a PR.
